keras/preprocessing/sequence.py

```diff
@@ -216,3 +218,93 @@ def _remove_long_seq(maxlen, seq, label):
             new_seq.append(x)
             new_label.append(y)
     return new_seq, new_label
+
+
+class TimeSeriesSequence(Sequence):
+    """Utility class for generating batches of temporal data.
+
+    This class takes in a sequence of data-points gathered at 
+    equal intervals, along with time series parameters such as 
+    stride, delay etc., to produce batches for training/validation.
+
+    # Arguments
+        data: indexable generator (such as list) containing
```

Comment:
It isn't clear what an "indexable generator" is in this context. "Such as a list or a Numpy array"?